### PR TITLE
Refactor ENF version check

### PR DESCRIPTION
**Pull request by d4rken**

### CODE REVIEW

**Review by chris-cwa — APPROVED** [review]
looks clean

**Comment by BMItter** [comment]
still checking..

**Comment by d4rken** [comment]
Sorry, let me know if you find a bug will create follow up PR.

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/nearby/modules/diagnosiskeyprovider/DefaultDiagnosisKeyProvider.kt [hunk 1]
```diff
@@ -25,13 +25,9 @@ class DefaultDiagnosisKeyProvider @Inject constructor(
...
-        // Check version of ENF
-        try {
-            enfVersion.requireAtLeast(ENFVersion.V16)
-        } catch (e: Exception) {
-            Timber.e(e)
-            throw e
-        }
+        // Check version of ENF, WindowMode since v1.5, but version check since v1.6
+        // Will throw if requirement is not satisfied
+        enfVersion.requireMinimumVersion(ENFVersion.V1_6)
```
Comment (BMItter) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/nearby/modules/diagnosiskeyprovider/DefaultDiagnosisKeyProvider.kt:30]: here I miss some kind of log before . (checking version... or something).
(no showstopper)

**Pull request merged**